cpp/examples/primal_dual/inpainting.cc

```diff
@@ -16,7 +16,6 @@
 #include "sopt/power_method.h"
 #include "sopt/relative_variation.h"
 #include "sopt/sampling.h"
-#include "sopt/types.h"
```

Comment:
Why this removal? We are using `Vector`, `Matrix` and `Image` defined in `types.h` so I think we should explicitly include it, even though it is probably being included by some of the other headers.

Reply:
types.h was already included in the file, and this second inclusion is a duplicate.

Reply:
I see! Sorry I missed that